website/src/pages/index.tsx

```diff
@@ -9,13 +9,57 @@ import { EnterpriseReadySolution } from '../components/product/EnterpriseReadySo
 import { IntegratesWithSection } from '../components/product/IntegratesWithSection'
 import { ProductDemoVideo } from '../components/product/ProductDemoVideo'
 import { ProductFeaturesAndUseCases } from '../components/product/ProductFeaturesAndUseCases'
-import { Testimonials } from '../components/Testimonials'
+import { Tweets } from '../components/Tweets'
 import { Vimeo } from '../components/Vimeo'
 import { ContactPresalesSupportAction } from '../css/components/actions/ContactPresalesSupportAction'
 import { GetSourcegraphNowActions } from '../css/components/actions/GetSourcegraphNowActions'
 import { RequestDemoAction } from '../css/components/actions/RequestDemoAction'
 import { ViewDeveloperDocumentationAction } from '../css/components/actions/ViewDeveloperDocumentationAction'
 
+const testimonials: Testimonial[] = [
+    {
+        customer: 'Yelp',
+        logo: '/external-logos/yelp.svg',
+        quote: 'Sourcegraph empowers developers at Yelp to ship code faster and more reliably than ever before.',
+        author: {
+            name: 'Yelp Software Engineers: Mark Larah, Dennis Coldwell, and Kevin Chen',
+        },
+        cta: {
+            text: 'Read engineering blog post',
```

Comment:
Can this also just be "Read case study"? I would think that it'd sound more compelling than "engineering blog post" for a decision maker, and it probably doesn't matter that it takes them to Yelp's blog.

Reply:
Sounds good!